test/browser/spec/ol/interaction/Draw.test.js

```diff
@@ -1686,6 +1686,48 @@ describe('ol/interaction/Draw', function () {
     });
   });
 
+  describe('#setType()', function () {
+    let interaction;
+
+    beforeEach(function () {
+      interaction = new Draw({
+        type: 'LineString',
+      });
+
+      expect(interaction.getActive()).to.be(true);
+
+      map.addInteraction(interaction);
+
+      // first point
+      simulateEvent('pointermove', 10, 20);
+      simulateEvent('pointerdown', 10, 20);
+      simulateEvent('pointerup', 10, 20);
+
+      expect(interaction.sketchFeature_).not.to.be(null);
+    });
+
+    afterEach(function () {
+      map.removeInteraction(interaction);
+    });
+
+    describe('#setType()', function () {
+      it('sets type property', function () {
+        interaction.setType('Polygon');
+        expect(interaction.getType()).to.be('Polygon');
+
+        interaction.setType('Point');
+        expect(interaction.getType()).to.be('Point');
```

Comment:
Please add `expect` calls for the other values that are updated by the call to `setType`. (mode and min/max point)